storage/buckets.rb

```diff
@@ -90,6 +90,78 @@ def create_bucket project_id:, bucket_name:
   # [END create_bucket]
 end
 
+def create_bucket_class_location project_id:, bucket_name:
+  # [START create_bucket_class_location]
+  # project_id    = "Your Google Cloud project ID"
+  # bucket_name   = "Name of Google Cloud Storage bucket to create"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
+  bucket  = storage.create_bucket bucket_name,
+                                  location:      "ASIA",
+                                  storage_class: "COLDLINE"
+
+  puts "Created bucket #{bucket.name}"
+  # [END create_bucket_class_location]
+end
+
+def list_bucket_labels project_id:, bucket_name:
+  # [START get_bucket_labels]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
+  bucket  = storage.bucket bucket_name
+
+  puts "Updated labels for #{bucket_name}"
+  bucket.labels.each do |key, value|
+    puts "#{key} = #{value}"
+  end
+  # [END get_bucket_labels]
+end
+
+def add_bucket_label project_id:, bucket_name:, label_key:, label_value:
+  # [START add_bucket_label]
+  # project_id  = "Your Google Cloud project ID"
+  # bucket_name = "Name of your Google Cloud Storage bucket"
+  # label_key   = "Cloud Storage bucket Label Key"
+  # label_value = "Cloud Storage bucket Label Value"
+
+  require "google/cloud/storage"
+
+  storage = Google::Cloud::Storage.new project: project_id
+  bucket  = storage.bucket bucket_name
+
+  bucket.update do |bucket|
+    bucket.labels[label_key] = label_value
+  end
+
+  puts "Added label to #{bucket_name}"
```

Comment:
Reading back through this... now I'm thinking that having the output include the key and value would be nice and clear. I think I argued to make this simpler, but it's maybe slightly too simple.

`puts "Added label #{label_key} with value #{label_value} to #{bucket_name}"`

Thoughts?

Reply:
As a user it'd nice to print key and value, but a user could also list labels for a bucket after making a new label to verify.

Reply:
Per previous comments, I really don't think it makes sense to **list** the labels in an **add** or **remove** sample.

Listing samples is substantial enough of a task that it has *its own sample snippet*

For samples, we print information about the operation. We do not **prove** that it works.

What we [create a Spanner database](https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/f2b32b69406a7c8af36bf9f5f06441c380ac9260/spanner/spanner_samples.rb#L46) we don't print out all Spanner databases to prove that the operation was successful.

Reply:
But when we create a Spanner database, we do show the name of the database we added and what instance

```ruby
puts "Created database #{database_id} on instance #{instance_id}"
```

That's why I think it would be appropriate to include the key and value.

It's more comprehensive IMO and consistent with other samples.

Reply:
Gotcha, I'll update the sample to include key and value. Consistency!

Reply:
🤘